unit-tests/live/intrinsics/test-motion-intrinsics.py

```diff
@@ -0,0 +1,33 @@
+# License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
Please rename the file to `test-motion.py`
Currently when it runs it looks like this
intrinsics-motion-intrinsics

Reply:
I will do it in the end...